### PR TITLE
Fix -fPIC error and metas configuration

### PR DESCRIPTION
**Pull request by julibert**
This pull request fixes issue #57.

### CODE REVIEW

**Comment by iluetkeb** [comment]
How does this address the issue? I see only "isolated build on"?

File: micro_ros_setup/config/host/generic/client-host-colcon.meta [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
     "names":{
+        "*":{
```
Comment (pablogs9) [inline micro_ros_setup/config/host/generic/client-host-colcon.meta]: Is this wildcard documented in colcon .meta files?

Comment (julibert) [reply micro_ros_setup/config/host/generic/client-host-colcon.meta]: I didn't find documentation in colcon but I saw this here https://github.com/micro-ROS/micro-ros-build/blob/dashing/micro_ros_setup/config/agent-colcon.meta

**Comment by julibert** [comment]
I rebased it over `feature/colcon_pkg` branch.

**Comment by BorjaOuterelo** [comment]
> Looks good to me.
> 
> The freertos failure seems to have been the toolchain download only.

Right, toolchain download in FreeRTOS is causing random failures, not sure how to overcome those TBH.

**Pull request merged**